ui-library-add-call-recording/api/host.json

```diff
@@ -1,15 +1,11 @@
-{
-  "version": "2.0",
-  "logging": {
-    "applicationInsights": {
-      "samplingSettings": {
-        "isEnabled": true,
-        "excludedTypes": "Request"
-      }
-    }
-  },
-  "extensionBundle": {
-    "id": "Microsoft.Azure.Functions.ExtensionBundle",
-    "version": "[2.*, 3.0.0)"
-  }
-}
+{
+    "version": "2.0",

```

Comment:
i am curious if we want to mention applicationInsights here. Should we just delete it to be cleaner?

Reply:
This is the default template generated by vscode for azure functions. Will look at removing this.